frotz/src/dumb/dumb_init.c

```diff
@@ -23,7 +23,7 @@
 #include "dumb_frotz.h"
 #include "dumb_blorb.h"
 
-f_setup_t f_setup;
+extern f_setup_t f_setup;
```

Comment:
I think this one shouldn't be `extern`. According to your log in issue #44 
> /usr/bin/ld: src/dumb/dumb_input.o:(.bss+0x0): multiple definition of `f_setup'; src/dumb/dumb_init.o:(.bss+0x0): first defined here

`dumb_init.o` seems to contain the first definition.

Reply:
PS: it makes `pytest tests` happy on my machine.

Reply:
Yes I just realized the same thing, no complaints on my machine also. Commit made.